src/base/rss/rss_feed.cpp

```diff
@@ -424,6 +410,85 @@ void Feed::downloadIcon()
             , this, &Feed::handleIconDownloadFinished);
 }
 
+int Feed::updateArticles(QList<QVariantHash> loadedArticles)
```

Comment:
seems turning `loadedArticles` into const reference is possible

Reply:
Comments are addressed.